lib/cf-socket.c

```diff
@@ -1602,6 +1609,11 @@ CURLcode Curl_cf_tcp_create(struct Curl_cfilter **pcf,
     result = CURLE_OUT_OF_MEMORY;
     goto out;
   }
+
```

Comment:
detail, just to be consistent with the rest of the code: either also add a new line after `#endif` or no new line here.